controllers/ovndbcluster_controller.go

```diff
@@ -579,8 +571,117 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 			if err != nil {
 				return ctrl.Result{}, err
 			}
+			// Delete DNS records for deleted services/pods
+			dnsData := &infranetworkv1.DNSData{}
+			dnsName := fmt.Sprintf("dns-%s-%d", serviceName, i)
```

Comment:
you duplicate the construction of the dnsName here and below. Please introduce a const / helper to generate the record name.

Reply:
I also wonder if we actually need the dns- prefix. It's DnsData, it's clear it's DNS, no?

Reply:
Yeah, I was following the naming of other items. Before this PR DNSData was used for svc, which resolves the different services:
```
spec:
  dnsDataLabelSelectorValue: dnsdata
  hosts:
  - hostnames:
    - barbican-internal.openstack.svc
    - cinder-internal.openstack.svc
    - glance-default-internal.openstack.svc
    - keystone-internal.openstack.svc
    - neutron-internal.openstack.svc
    - nova-internal.openstack.svc
    - nova-metadata-internal.openstack.svc
    - placement-internal.openstack.svc
    - swift-internal.openstack.svc
    ip: 172.17.0.80
  - hostnames:
    - rabbitmq.openstack.svc
    ip: 172.17.0.85
  - hostnames:
    - rabbitmq-cell1.openstack.svc
    ip: 172.17.0.86
```
This DataStructure is called `dns-svc`. That's why my structure starts also with dns-.
On the other hand, once the edpm node is configured it creates another DNSData object, which is `openstack-edpm`, where it defines the edpm-compute-N.NETWORK_NAME.example.com.

I'm okay with keeping/removing the DNS.

Reply:
I'm ok if you leave the prefix. Please construct the name with a helper / format const.

Reply:
Done